frontend/src/metabase/query_builder/components/CompareAggregations/CompareAggregations.tsx

```diff
@@ -67,7 +72,49 @@ export const CompareAggregations = ({
         columns,
         offset,
       );
-      onSubmit(aggregations);
+
+      if (!columnAndBucket) {
+        return;
+      }
+
+      let nextQuery = aggregations.reduce(
+        (query, aggregation) => Lib.aggregate(query, stageIndex, aggregation),
+        query,
+      );
+      let breakout = columnAndBucket.breakout;
+
+      const column = Lib.withTemporalBucket(
+        columnAndBucket.column,
+        columnAndBucket.bucket,
+      );
+
+      if (breakout) {
+        // replace the breakout
+        const breakoutIndex = Lib.breakouts(nextQuery, stageIndex).indexOf(
```

Comment:
This is not correct! There is no guarantee if would ever work. It works only because of memoization.

Reply:
Please get rid of this